.dotnet/tests/Models/ModelsMockTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.ClientModel;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenAI.Models;
+using OpenAI.Tests.Utility;
+
+namespace OpenAI.Tests.Models;
+
+[TestFixture(true)]
+[TestFixture(false)]
+[Parallelizable(ParallelScope.All)]
+[Category("Smoke")]
+public partial class ModelsMockTests : SyncAsyncTestBase
```

Comment:
I noticed `ModelClient` methods do not support passing a `CancellationToken`. Is this intentional?

Reply:
Good catch! I have added this to our backlog for GA.